mockk/common/src/main/kotlin/io/mockk/impl/stub/MockKStub.kt

```diff
@@ -32,13 +33,15 @@ open class MockKStub(
     override fun addAnswer(matcher: InvocationMatcher, answer: Answer<*>) {
         val invocationAnswer = InvocationAnswer(matcher, answer)
         answers.add(invocationAnswer)
+        answerUsages[invocationAnswer] = 0
     }
 
     override fun answer(invocation: Invocation): Any? {
         val invocationAndMatcher = InternalPlatform.synchronized(answers) {
             answers
                 .reversed()
                 .firstOrNull { it.matcher.match(invocation) }
+                ?.also { answerUsages[it] = answerUsages[it]!! + 1 }
```

Comment:
How about using `answerUsages.getOrDefault(it, 0)` here rather than the !!?

I know `answerUsages[it]` is definitely not null because you initialize it in `addAnswer`, but `getOrDefault` seems safer to me.